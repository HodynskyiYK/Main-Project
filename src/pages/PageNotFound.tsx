import React, { FC } from 'react'

export const PageNotFound: FC = () => {

    return (
        <div>
            <h1>Page Not Found</h1>
        </div>
    )
}
