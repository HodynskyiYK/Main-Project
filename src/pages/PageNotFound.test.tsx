import React from 'react'
import { render } from '@testing-library/react'
import { PageNotFound } from './PageNotFound'

describe('PageNotFound tests', () => {

    test('To match snapshot', () => {
        const {asFragment} = render(<PageNotFound/>)
        expect(asFragment()).toMatchSnapshot()
    })
})