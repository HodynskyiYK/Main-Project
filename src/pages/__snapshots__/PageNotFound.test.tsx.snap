// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound tests To match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Page Not Found
    </h1>
  </div>
</DocumentFragment>
`;
